### PR TITLE
Add Bandcamp provider 

### PR DESCRIPTION
**Pull request by mickythump**

### CODE REVIEW

File: providers/bandcamp/bandcamp.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("bandcamp", &Bandcamp{})
+}
+
+// BaiduProvider adheres to the Provider interface.
+type Bandcamp struct {
```
Comment (zquestz) [inline providers/bandcamp/bandcamp.go]: Should be BandcampProvider

File: providers/atmospherejs/atmospherejs.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("atmospherejs", &AtmosphereJS{})
+}
+
+// BaiduProvider adheres to the Provider interface.
```
Comment (zquestz) [inline providers/atmospherejs/atmospherejs.go]: Not Baidu =)

**Comment by zquestz** [comment]
Probably remove atmosphere provider, as there is a separate PR for that.

**Comment by zquestz** [comment]
Mind rebasing this PR to 1 commit? Looks good and is ready to merge after that.

**Comment by mickythump** [comment]
I've squashed it to one commit before mine. I hope it's not a big deal. If it matters, you can close the PR, and I'll make another 'clean' one.

**Comment by zquestz** [comment]
Sorry to make you recreate. But we need it to not be on the old commit.

**Pull request closed without merging**